examples/AspNetCore/Startup.cs

```diff
@@ -147,6 +147,8 @@ public void ConfigureServices(IServiceCollection services)
                         break;
                 }
             });
+
+            services.Configure<PrometheusExporterOptions>(this.Configuration.GetSection("Prometheus"));
```

Comment:
Could move this up into the switch (~line 132). Might better capture how `PrometheusExporterOptions` is related to `AddPrometheusExporter`.